torch/_tensor_docs.py

```diff
@@ -2848,15 +2848,26 @@ def callable(a, b) -> number
 
 add_docstr_all('size',
                r"""
-size() -> torch.Size
+size(dim) -> torch.Size or int
```

Comment:
can we document this as (kind of) two separate functions, a-la https://pytorch.org/docs/master/generated/torch.mean.html?